src/api/app/views/webui2/webui/user/_involvement.html.haml

```diff
@@ -1 +1,75 @@
-Placeholder for things the user is involved in
+.card.mb-3
+  .bg-light
+    %ul.nav.nav-tabs.pt-2.px-3.flex-nowrap.collapsible{ 'role': 'tablist' }
```

Comment:
We are missing the dropdown for this correctly collapse. Refer to the [patterns here](https://obs-patterns.netlify.com/atoms/tabs.html)